docs/people-and-groups/api-keys.md

```diff
@@ -0,0 +1,110 @@
+---
+title: API keys
+---
+
+# API keys
+
+Metabase can create API keys to authenticate programmatic requests to the API.
+
+## Fair warning about Metabase's API
+
+We don't version the [Metabase API](../api-documentation.md). We rarely change API endpoints, and almost never remove them, but if you write code that relies on the API, there's a chance you might have to update your code in the future.
+
+That said, there are times when it's nice to work with the API, like when managing permissions with a large number of people and groups, or bulk archiving, or content creation. So we added the ability to create API keys to authenticate your programmatic requests.
+
+## Create an API key
+
+To create an API key:
+
+1. Click on the **gear** icon in the upper right.
+2. Select **Admin settings**.
+3. Go to the **Settings** tab.
+4. Click on the **Authentication** tab on the left menu.
+5. Scroll to **API Keys** and click **Manage**.
+6. Click the **Create API Key** button.
+7. Enter a **Key name**. You can have multiple API keys, so give it a name that will help you remember what you're using the key for.
+8. Select a **Group**. The key will have the same permissions granted to that group.
+9. Click **Create**.
+10. Copy the generated API key and save it somewhere safe. Metabase won't be able to show you the key again. If you lose the key, you'll need to generate a new key.
+
+## Managing API Keys
+
+To view and manage existing API keys:
+
+1. Click on the **gear** icon in the upper right.
+2. Select **Admin settings**.
+3. Go to the **Settings** tab.
+4. Click on the **Authentication** tab on the left menu.
+5. Scroll to **API Keys** and click **Manage**.
+
+### Editing API keys
+
+To edit an API key, scroll to the key you want to edit and click on the **pencil** icon. Metabase will pop up an **Edit API Key** modal where you can edit:
+
+- The key's name
+- Which group the key belongs to.
+- Change (regenerate) the key. Metabase will replace the existing API key with a new key. You won't be able to recover the old key.
+
+### Deleting API keys
+
+You won't be able to recover a deleted API key. You'll have to create a new key.
+
+To delete an API Key:
+
+1. Click on the **gear** icon in the upper right.
+2. Select **Admin settings**.
+3. Go to the **Settings** tab.
+4. Click on the **Authentication** tab on the left menu.
+5. Scroll to **API Keys** and click **Manage**.
+6. Select the key you want to delete and click the **trash** icon.
+7. Metabase will pop up a **Delete API Key** modal. Click the **Delete API Key** button.
+
```

Comment:
I'd add something here about what happens if the group the key belongs to is deleted.

If the group is removed or deleted, the key is automatically transferred to the "All users" group.